docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,171 @@
+---
+title: ".NET Docs What's New (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 2019/12/13
+---
+
+# .NET Docs What's New (November 2019)
+
+Welcome to what's new in .NET docs for November 2019. Listed below are some of the recent docs we've published.
+
+## .NET Core
+
+- [Breaking changes, version 2.2 to 3.0 - .NET Core](../docs/core/compatibility/2.2-3.0.md)
+- [Breaking changes, version 2.2 to 3.1 - .NET Core](../docs/core/compatibility/2.2-3.1.md)
+- [dotnet-trace tool - .NET Core](../docs/core/diagnostics/dotnet-trace.md)
+- [.NET Core SDK and runtime dependencies - .NET Core](../docs/core/install/dependencies.md)
+- [Check installed .NET Core versions on Windows, Linux, and macOS - .NET Core](../docs/core/install/how-to-detect-installed-versions.md)
+- [https://raw.githubusercontent.com/dotnet/docs/master/docs/core/install/includes/package-manager-heading-hack-pkgname.md](../docs/core/install/includes/package-manager-heading-hack-pkgname.md)
```

Comment:
Why does this have a URL in the link text?

Reply:
That was a bug. It's fixed now.